Readme.md

```diff
@@ -150,6 +150,9 @@ Please go through the Contributing guidelines <a href="https://github.com/FOSS-C
 ## 📖 Code Of Conduct
 You can find the Code of Conduct <a href="https://github.com/FOSS-Cell-GECPKD/pixelvibe/blob/main/CODE_OF_CONDUCT.md">here</a>.
 
+### ✅&nbsp; PEP8 Standards to be followed
+-	Please follow the guidelines of PEP8 as gevin in [here]()
```

Comment:
1. It should be `given` not `gevin`.
2. Also add information about python module `pycodestyle`.
3. Update the link `[here]()`